injected/src/utils.js

```diff
@@ -22,25 +21,14 @@ export function getInjectionElement() {
     return document.head || document.documentElement;
 }
 
-// Tests don't define this variable so fallback to behave like chrome
-const hasMozProxies = typeof mozProxies !== 'undefined' ? mozProxies : false;
-
 /**
  * Creates a script element with the given code to avoid Firefox CSP restrictions.
  * @param {string} css
  * @returns {HTMLLinkElement | HTMLStyleElement}
  */
 export function createStyleElement(css) {
-    let style;
-    if (hasMozProxies) {
-        style = document.createElement('link');
-        style.href = 'data:text/css,' + encodeURIComponent(css);
-        style.setAttribute('rel', 'stylesheet');
-        style.setAttribute('type', 'text/css');
-    } else {
-        style = document.createElement('style');
-        style.innerText = css;
-    }
+    const style = document.createElement('style');
```

Comment:
Did you check these style elements injected against strict CSP sites? (If it works we should change the function description comment).

I think it'd be probably safer to keep this in, instead checking for the platform name instead.